### PR TITLE
Changed page title

### PR DESCRIPTION
**Pull request by ahboujelben**

### CODE REVIEW

**Comment by ahboujelben** [comment]
Current dependencies on/for this PR:
* **#6** <a href="https://app.graphite.dev/github/pr/wee-programs/graphite-playground/6?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/wee-programs/graphite-playground/6?utm_source=stack-comment).

**Review by ahboujelben — COMMENTED** [review]
First PR!

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>Graphite Playground!</title>
```
Comment (KacperKozak) [inline index.html:7]:
```suggestion
    <title>Graphite Playground</title>
```

Comment (ahboujelben) [reply index.html]: Done

**Review by KacperKozak — APPROVED** [review]

**Comment by ahboujelben** [comment]
### Merge activity

* **Mar 12, 5:49 AM EDT**: @ahboujelben merged this pull request with [Graphite](https://app.graphite.dev/github/pr/wee-programs/graphite-playground/6).

**Pull request merged**